nix/default.nix

```diff
@@ -28,11 +62,48 @@ let
     passthru = {
       inherit python;
 
-      docker = pkgs.dockerTools.buildLayeredImage {
+      docker = pkgs.dockerTools.buildImage {
+        runAsRoot = ''
+          #!${pkgs.stdenv.shell}
+          ${pkgs.dockerTools.shadowSetup}
+          groupadd --gid 10001 app
+          useradd --gid 10001 --uid 10001 --home-dir /app app
+          mkdir -p --mode=1777 /tmp
+          mkdir -p -m 700 /app
+          chown app:app /app
+        '';
         name = "shipitscript";
         tag = version;
+        config = {
+          Healthcheck = {
```

Comment:
I think we should just put the file at the root like we discussed at the meeting since cloudops is not looking at this configuration.

Reply:
I put this into `runAsRoot` and linked as `/bin/healthcheck`